src/Paramore.Brighter.MsSql.EntityFrameworkCore/Paramore.Brighter.MsSql.EntityFrameworkCore.csproj

```diff
@@ -1,22 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-      <Description>TThis is the MsSql Connection provider to obtain the connection from Entity Framework Core.</Description>
-      <Authors>Paul Reardon</Authors>
-      <TargetFrameworks>netstandard2.1;net6.0</TargetFrameworks>
-      <PackageTags>RabbitMQ;AMQP;Command;Event;Service Activator;Decoupled;Invocation;Messaging;Remote;Command Dispatcher;Command Processor;Request;Service;Task Queue;Work Queue;Retry;Circuit Breaker;Availability</PackageTags>
-    </PropertyGroup>
+  <PropertyGroup>
+    <Description>TThis is the MsSql Connection provider to obtain the connection from Entity Framework Core.</Description>
+    <Authors>Paul Reardon</Authors>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
```

Comment:
Do we need netstandard2.1? If we can be netstandard2.0 I would have thought that netstandard2.1 is not required. We could just choose netstandard2.0 and net6.0

Reply:
good point, totally agree... adding the commit.